CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.0.0 - May 4, 2020
+
+**BREAKING CHANGES** 🚨
+
+- Remove bundled calendars see [this pr](https://github.com/gocardless/business/pull/54)
```

Comment:
I'd add a second sentence here pointing to the old data in Github.

Potentially a second in the README along the lines of "Hey, how do I unbreak now you've unbundled the data?"

Reply:
Will do!

Reply:
maybe with a message along the lines of "this data only goes until 2020, after that you're on your own"